### ISSUE TITLE
Publish container 

### ISSUE THREAD

**Issue by AstraLuma** [zorchenhimer/MovieNight#186]
Build and publish a first-party container image.

I recently did this for another project with GitHub Actions and GitHub container registry. I can make a pr later if you want.

**Comment by CptPie** [comment]
To give you an update on this issue:

We discussed this internally in the discord server (link in the about section of this project).
This project already has some docker build files which were contributed a while ago but we are missing anyone maintaining them, several people came around asking questions regarding the docker deployment and none of us could properly help.

While nobody here is opposed to supplying docker containers, we worry about the maintenance (especially since sometimes in the future :tm: this project might switch to hls streaming which would require a major rework of the containers).

TLDR: If you would commit long term to maintaining the docker containers (and fixing the current docker stuff) i don't think there is any reason to oppose your offer of an PR.

**Comment by zorglube** [comment]
Since I use a Docker deployed version of Movie Night, I'm interested in maintaining the Docker file.... After learning more about Docker.

**Comment by AstraLuma** [comment]
As someone that's more DevOps than go developer, that would definitely be appreciated.

**Comment by joeyak** [comment]
This was initially done in ea2b6ef1e77c5d584e00e1a314e72cb5d11f0475
